### DISCUSSION TITLE
Whitelisting the clients for API Consumption

### DISCUSSION THREAD

**Discussion by psidana1983** [camaraproject/WorkingGroups#243] [Q&A]
Hi Team , 
In our organization we need to whitelist any new client we get .So wanted to check does CAMARA promote this or CAMARA doesn't promote any manual interaction with client , as it is required in my organization . 

Regards
Prashant Sidana

**Comment by eric-murray** [comment]
At the moment CAMARA APIs use OAuth2 to authenticate API clients, but that may change in future. See [CAMARA-AuthN-AuthZ-Concept.md](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/CAMARA-AuthN-AuthZ-Concept.md) and [API-design-guidelines.md](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-design-guidelines.md).
